cognite/client/_api_client.py

```diff
@@ -104,6 +104,7 @@ class APIClient:
                     "hostedextractors/.*",
                     "postgresgateway/.*",
                     "context/diagram/.*",
+                    "ai/tools/documents/(summarize|ask)",
```

Comment:
Please add a test that is "retryable" and one that shouldn't be to the `parametrize` of `test_is_retryable`